src/runtime/manifest-parser.pegjs

```diff
@@ -93,7 +93,11 @@
   }
 
   function toAstNode<T extends {location: IFileRange} & Omit<T, 'location'>>(data: Omit<T, 'location'>): T {
-    return {...data, location: location()} as T;
+    const loc = location();
+    // For debugging:
+    // const txt = text();
+    // loc['text'] = txt;
```

Comment:
If this is generally useful I think we can just always do it; the manifest parser isn't a performance-critical component.

Otherwise I think we should remove; generally it's not good to have commented out code checked in.

Reply:
Added, yeah, it's helpful for debugging + might be handy for other things later.

I considered putting it behind a flag, but as you said, it's not perf critical.